advocacy_docs/partner_docs/ThalesGuide/02-PartnerInformation.mdx

```diff
@@ -1,12 +1,13 @@
 ---
-title: 'Partner Information'
+title: '1. Partner Information'
```

Comment:
Remove the numbers from the titles in all the files. Numbering isn't necessary in online content and numbering is automatically added to the PDFs, I believe).